Src/BootCamp.Chapter/Item.cs

```diff
@@ -3,18 +3,24 @@
     public class Item
     {
         private string _name;
+
         public string GetName()
         {
             return _name;
         }
 
         private decimal _price;
+
         public decimal GetPrice()
         {
             return _price;
         }
 
         private float _weight;
+        public float GetWeight()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetWeight' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5138976)